File: CSES/1-Introductory-Problems/1-Weird-Algorithm/main1.c
#include <stdio.h>

int main()
{
    //Variables
    int n = 1;

    //Input
    scanf("%d",&n);

    //Calculation and Output
    printf("%d ",n);
    while (n != 1)
    {
        if (n%2) n = (n*3)+1;
        else n /=2;
        printf("%d ",n);
    }

    return 0;
}

File: Quera/1-Programming-basics/easy/014-Diamond-Printing/Main1.java
import java.util.Scanner;

public class Main1 {

    public static void main(String[] args) {

        //Variables
        int n;


        //Input
        Scanner getScanner = new Scanner(System.in);

        n = getScanner.nextInt();


        //Calculation & Output
        for (int i = 0; i <= n; i++) {
            for (int j = n-i; j > 0; j--) {
                System.out.print(" ");
            }
            for (int j = 0; j < (i*2)+1; j++) {
                System.out.print("*");
            }
            System.out.print("\n");
        }

        for (int i = n-1; i >= 0; i--) {
            for (int j = n-i; j > 0; j--) {
                System.out.print(" ");
            }
            for (int j = 0; j < (i*2)+1; j++) {
                System.out.print("*");
            }
            System.out.print("\n");
        }

        getScanner.close();
    }
}
